manifests/install.pp

```diff
@@ -113,8 +113,9 @@
             }
           }
 
+          $fail_restart_command = 'SC.exe failure Docker reset= 432000 actions= restart/30000/restart/60000/restart/60000'
```

Comment:
This is potentially an unneeded addition given that there is no interpolation.

Reply:
Whoops, must have slipped while I was in zombie mode working on those commands. Will undo.